_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -302,6 +302,23 @@ uint32_t ConvertRateControlMFX2VAAPI(mfxU8 rateControl, bool bSWBRC)
     }
 }
 
+mfxU8 ConvertSliceStructureVAAPIToMFX(mfxU8 structure)
```

Comment:
`static`

Why this is in H.265 only? Looks to be good generic utils function....